### ISSUE TITLE
Dataization of object without `@` attribute 

### ISSUE THREAD

**Issue by Chamber6821** [objectionary/eo#3119]
Description of atom `dataized` does bot explain how this object will be dataized:

```
[] > obj
  1 > one
  2 > two
  3 > tree
```

**Comment by maxonfjvipon** [comment]
@Chamber6821 it's not really responsibility of object `dataized`. Dataization is the process of taking `Δ` attribute from the object. "Taking an attribute" is also called "dynamic dispatch" or "dot notation". [Here](https://arxiv.org/abs/2111.13384) should be described how it works, and also how decorators work. There should be all possible cases:
1. attribute is present in the object
2. attribute is absent, but `@` attribute is present
3. attribute is absent, but object is an atom
4. attribute is absent and object is not an atom and `@` attribute is absent

If there are not, let us know, we'll improve it

**Comment by yegor256** [comment]
@maxonfjvipon @Chamber6821 it seems that "dataization" is confused with "serialization" here. Maybe we should explicitly in the doc block of the `dataized` object that it doesn't serialize an object?

**Comment by maxonfjvipon** [comment]
@yegor256 what is serialization in terms of EO?

**Comment by yegor256** [comment]
@maxonfjvipon there is no such thing in EO, but people get confused since they believe there is.

**Comment by Chamber6821** [comment]
@yegor256 yes, I do not understand what difference between `dataozation` and `serialisation` and the description or tests does not helping me to understand difference. I think difference between `dataization` and `serialisation` should be explain via small tests and description should pay attention to tests.

Now tests does not explain next edge cases:
1. What happens with objects without `@` attribute?
2. What happens with objects with infinity recursion in definition of `@`?
3. What happens with object that decorates object without `@`?

It is also worth paying attention in the description that dataization is not serialisation and why. Explain main difference between them. Now `dataized` looks like wrapper that "call" and save `obj.as- bytes`.

**Comment by yegor256** [comment]
@maxonfjvipon maybe we can write a small blog post about the `dataized` object, with examples?

**Comment by maxonfjvipon** [comment]
@yegor256 I think we need a blog post which explains what is dataization at all

**Comment by Chamber6821** [comment]
When will I be able to read the post?
